scripts/app.js

```diff
@@ -1,62 +1,60 @@
 'use strict';
 
-var images = [];
+var app = app || {};
 
-function Project (rawDataObj) {
-  for (var key in rawDataObj) {
-    this[key] = rawDataObj[key];
+(function(module) {
+  function Project (rawDataObj) {
+    for (var key in rawDataObj) {
```

Comment:
Dear Liz,

I don't know if there is a way I could replace this for loop. I don't think so. Not today, satan!

Love, Izzy